src/layman_settings.py

```diff
@@ -84,10 +84,10 @@
 LAYMAN_GS_WMS_WORKSPACE_POSTFIX = '_wms'
 
 LAYMAN_OUTPUT_SRS_LIST = [
-    int(code) for code in os.environ['LAYMAN_OUTPUT_SRS_LIST'].split(',')
+    f'EPSG:{int(code)}' for code in os.environ['LAYMAN_OUTPUT_SRS_LIST'].split(',')
```

Comment:
I like using int() here, because it raises exception if non-integer string is used in environment variable, where it could happen easily.